packages/anoncreds-rs/src/services/AnonCredsRsHolderService.ts

```diff
@@ -55,19 +55,9 @@ export class AnonCredsRsHolderService implements AnonCredsHolderService {
     agentContext: AgentContext,
     options?: CreateLinkSecretOptions
   ): Promise<CreateLinkSecretReturn> {
-    let masterSecret: MasterSecret | undefined
-    try {
-      masterSecret = MasterSecret.create()
-
-      // FIXME: This is a very specific format of anoncreds-rs. I think it should be simply a string
-      const linkSecretJson = masterSecret.toJson() as { value: { ms: string } }
-
-      return {
-        linkSecretId: options?.linkSecretId ?? utils.uuid(),
-        linkSecretValue: linkSecretJson.value.ms,
-      }
-    } finally {
-      masterSecret?.handle.clear()
+    return {
+      linkSecretId: options?.linkSecretId ?? utils.uuid(),
+      linkSecretValue: LinkSecret.create(),
```

Comment:
much better :)

Reply:
Yeah I was a really happy moment for me when I did that update :-D